extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -672,39 +692,35 @@ public static function isBlog( Title $title = null ) {
 	/**
 	 * editPage -- show edit form
 	 *
-	 * @access public
-	 *
 	 * @return String
 	 */
 	public function editPage() {
 		global $wgStylePath;
-		wfProfileIn( __METHOD__ );
 
-		$text = '';
-		$this->load( true );
-		if ( $this->canEdit() ) {
-			$vars = [
-				'canEdit'				=> $this->canEdit(),
-				'comment'				=> htmlentities( ArticleCommentsAjax::getConvertedContent( $this->mLastRevision->getText() ) ),
-				'isReadOnly'			=> wfReadOnly(),
-				'isMiniEditorEnabled'	=> ArticleComment::isMiniEditorEnabled(),
-				'stylePath'				=> $wgStylePath,
-				'articleId'				=> $this->mTitle->getArticleId(),
-				'articleFullUrl'		=> $this->mTitle->getFullUrl(),
-			];
-			$text = F::app()->getView( 'ArticleComments', 'Edit', $vars )->render();
+		if ( !$this->load( true ) ) {
```

Comment:
Actually check the return value of `load` and return empty string if it fails (the same return value used in the rest of this function for failure)